packages/affiliates/scripts/example-dapp-mining-calculator.js

```diff
@@ -0,0 +1,31 @@
+const { BigQuery } = require("@google-cloud/bigquery");
+const { getWeb3 } = require("@uma/common");
+const moment = require("moment");
+const { getAbi } = require("@uma/core");
+const { DappMining } = require("../libs/affiliates");
+const Queries = require("../libs/bigquery");
+
+const web3 = getWeb3();
+
+const empAddress = "0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5";
```

Comment:
Are these the same addresses from `affiliates/dappmining-config.example.js` and is there a way to link them programmatically so that you're not repeating hard coded vars?

Reply:
yea this is a great idea. I am going to import the example config file for the parameters.